src/main/java/com/dreammaster/bartworksHandler/BacteriaRegistry.java

```diff
@@ -185,20 +191,9 @@ private void runAdditionalFuelRecipes() {
                 .eut(TierEU.RECIPE_IV).addTo(sAutoclaveRecipes);
 
         // super heavy -> heavy radox conversion
-        GT_Values.RA.addCentrifugeRecipe(
-                null,
-                null,
-                RadoxSuperHeavy.getFluid(1000),
-                RadoxHeavy.getFluid(2000),
-                null,
-                null,
-                null,
-                null,
-                null,
-                null,
-                null,
-                60000,
-                BW_Util.getMachineVoltageFromTier(8));
+        GT_Values.RA.stdBuilder().noItemInputs().itemOutputs().fluidInputs(RadoxSuperHeavy.getFluid(1000))
+                .fluidOutputs(RadoxHeavy.getFluid(2000)).duration(50 * MINUTES).eut(TierEU.RECIPE_UV)
+                .addTo(sCentrifugeRecipes);
```

Comment:
Should add `.noOptimize()`

![image](https://github.com/GTNewHorizons/NewHorizonsCoreMod/assets/53441451/21cca89a-d2b3-4ccd-957e-603938c112b0)